src/utils.rs

```diff
@@ -42,8 +43,19 @@ pub fn get_rss_instance() -> RustSemsimian {
     ]);
 
     let assoc_predicate: HashSet<TermID> = HashSet::from(["biolink:has_phenotype".to_string()]);
+    let rss = Mutex::new(Some(RustSemsimian::new(None, predicates, None, db)));
 
-    let mut rss = RustSemsimian::new(None, predicates, None, db);
-    rss.pregenerate_cache(&assoc_predicate, &SearchTypeEnum::Flat);
-    rss
+    {
```

Comment:
this part would be most useful as its own utility function in Semsimian, along with line 46 above, along with a parameter for passing a specific predicate or predicate list (and the whole target DB, for that matter)

Reply:
Agreed. Once confirmed that this solution works and does not have unforeseen side-effects, we can implement this in `semsimian` thus making this just a simple function call.